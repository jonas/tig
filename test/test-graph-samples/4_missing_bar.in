commit A B C
    Commit A - Merge B and C
commit B D E
    Commit B - Merge D and E
commit C F
    Commit C after F
commit F G D
    Commit F - Merge G and D
commit D G
    Commit D after G
commit G E
    Commit G after E
commit E
    Commit E
