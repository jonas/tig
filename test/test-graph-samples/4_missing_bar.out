●─╮ Commit A - Merge B and C
●─│─╮ Commit B - Merge D and E
│ ● │ Commit C after F
│ ●─│─╮ Commit F - Merge G and D
●─│─│─╯ Commit D after G
●─╯ │ Commit G after E
◎───╯ Commit E
